common/src/main/java/com/wynntils/models/gear/type/SetInstance.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.gear.type;
+
+import com.wynntils.models.stats.type.StatType;
+import java.util.Map;
+
+public class SetInstance {
+    private SetInfo setInfo;
+    private Map<String, Boolean> activeItems;
+    private int wynnCount;
+
+    public SetInstance(SetInfo setInfo, Map<String, Boolean> activeItems, int wynnCount) {
+        this.setInfo = setInfo;
+        this.activeItems = activeItems;
+        this.wynnCount = wynnCount;
+    }
+
+    public void update(SetInfo setInfo, Map<String, Boolean> activeItems, int wynnCount) {
+        this.setInfo = setInfo;
+        this.activeItems = activeItems;
+        this.wynnCount = wynnCount;
+    }
```

Comment:
The method is unused now. In fact, this is now an immutable class (as it should be, like GearInstance is), so you can make this a record, and remove every method.